### PR TITLE
Added Topological sort and Dijkstra algorithm in Java language

### PR DESCRIPTION
**Pull request by Ayan-10**
Fixes part of #7 which is Topological sort in Java language

### CODE REVIEW

**Comment by shubhidua** [comment]
Hello @raksha009, I would like to contribute to topological sorting in java. Please assign me this issue.

**Review by Siddhant-K-code — CHANGES_REQUESTED** [review]
LGTM! 

Just, fix Merge Conflicts

File: Code/Java/Dijkstra.java [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        return min_index;
+    }
+    static void printDijkstra(int distance[], int V, int src)
+    {
+        for (int i = 0; i < V; i++)
+            System.out.println("Distance of vertex "+i +" from source "+src+" is "+ distance[i]);
+    }
+}
+/*
+3
```
Comment (raksha009) [inline Code/Java/Dijkstra.java:102]: Follow 
Test Cases
Input
Output

File: Code/Java/Dijkstra.java [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+Distance of vertex 0 from source 0 is 0
+Distance of vertex 1 from source 0 is 11
+Distance of vertex 2 from source 0 is 2
+Distance of vertex 3 from source 0 is 31
+Distance of vertex 4 from source 0 is 24
+Distance of vertex 5 from source 0 is 28
+
+    Time complexity : O(V^2) V = no. of Vertices
+
+    Space Complexity: O(V^2)
```
Comment (raksha009) [inline Code/Java/Dijkstra.java:145]: Specify V

File: Code/Java/Dijkstra.java [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+*/
```
Comment (raksha009) [inline Code/Java/Dijkstra.java:147]: Add a new line here

File: Code/Java/Topological_sort.java [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    5 1
+    4 0
+    3 1
+    1 3
+    3 2
+    Output 2 :
+    5 4 1 3 2 0
+    Time Complexity: O(V + E) – where V is the number of vertices and E is the number of edges.
+    Space Complexity: O(V)
+*/
```
Comment (raksha009) [inline Code/Java/Topological_sort.java:111]: Add a new line here

File: Code/Java/Topological_sort.java [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+    4 2
+    5 1
+    4 0
+    3 1
+    1 3
+    3 2
+    Output 2 :
+    5 4 1 3 2 0
+    Time Complexity: O(V + E) – where V is the number of vertices and E is the number of edges.
+    Space Complexity: O(V)
```
Comment (raksha009) [inline Code/Java/Topological_sort.java:110]: Specify v here too

**Review by raksha009 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**